traffic_ops/app/conf/cdn.conf

```diff
@@ -36,7 +36,6 @@
         "oauth_client_secret": "",
         "routing_blacklist": {
             "ignore_unknown_routes": false,
-            "perl_routes": [],
```

Comment:
Can you make a simlilar update to the ansible TO config templates?

Reply:
I believe postinstall.pl is replaced by postinstall.py in this PR: https://github.com/apache/trafficcontrol/pull/4763

Reply:
So, `postinstall` is unchanged. It even still calls `_postinstall` with the changes made in #4763. The postinstall.py is _added_ as an alternative only, but the default is to still use the Perl version.

Installation is still `yum install trops.rpm` and then run `postinstall`. configuration is mostly untouched, just some things don't have any purpose anymore. I guess the biggest change is that some scripts may expect that by setting their `PERL5LIB` path (or whatever it's called) that their dependencies will be satisfied, but that won't actually be true for new installations. A lot of those aren't used anymore afaik, and many others have active PRs rewriting them. Oh, osversion.cfg is gone, only osversions.json is supported now.

> Can you make a simlilar update to the ansible TO config templates?

I can try, but I don't know much about Ansible - I kind of assume removing the Perl service might just break it all in half.

Reply:
So, I can't find any reference to this configuration file option in the Ansible files:

```shell-session
$ grep -r 'perl_routes' ./infrastructure/ansible | wc -l
0
```

Reply:
You're right, it looks like it was added and will be removed before anyone uses it in https://github.com/jhg03a/trafficcontrol/blob/ansible.refactor/infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

Reply:
Good point though on the PERL5Lib setup though if you'd like to remove it from https://github.com/apache/trafficcontrol/blob/5397a4fb17a24c255d698ce9594f37562b531c89/infrastructure/ansible/roles/traffic_ops/tasks/traffic_ops.yml#L127.

Reply:
I think it's fine to leave that there. If it's an existing install that's where the libraries will already be, and if it's a new install I don't think that'll hurt anything.